Library/Homebrew/tap_constants.rb

```diff
@@ -5,6 +5,8 @@
 HOMEBREW_TAP_FORMULA_REGEX = %r{^([\w-]+)/([\w-]+)/([\w+-.@]+)$}.freeze
 # Match taps' casks, e.g. `someuser/sometap/somecask`
 HOMEBREW_TAP_CASK_REGEX = %r{^([\w-]+)/([\w-]+)/([a-z0-9\-_]+)$}.freeze
+# Match main cask taps' casks, e.g. `homebrew/cask/somecask` or `somecask`
+HOMEBREW_MAIN_TAP_CASK_REGEX = %r{^(homebrew/cask/)?[a-z0-9\-_]+$}.freeze
```

Comment:
Not sure if this is the best place to put this but I couldn't find anywhere else that made more sense.